### ISSUE TITLE
splash screen - minor issue

### ISSUE THREAD

**Issue by redgrapetech** [benlau/quickandroid#38]
I have observed a minor jerk in the position of logo in the splash screen of the example 
app `quickandroidexample`
I tried using the sample concept as described by you in 
https://medium.com/@benlaud/complete-guide-to-make-a-splash-screen-for-your-qml-android-application-567ca3bc70af in a separate app and faced the same issue.

I think even quickandroidexample uses the same concept.
on my investigation i feel that the issue is that 
the theme draws the splash screen once as:

`apptheme.xml `  is

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="AppTheme" parent="@android:style/Theme.DeviceDefault.Light.NoActionBar">
        <item name="android:windowBackground">@drawable/splash</item>
    </style>
</resources>
 ```

and the instruction in  `AndroidManifest.xml` also makes it draws  once because of  

```
  <!-- Splash screen -->
<meta-data android:name="android.app.splash_screen_drawable" 
android:resource="@drawable/splash"/>
  <!-- Splash screen -->
```
And both of them do not place the logo at *exactly* the same place.

Thanks for  quickandroid .

regds
Rajesh Kumar Mallah.

**Comment by benlau** [comment]
yes, I also find this issue. I didn't go in deep to find out the main reason.  But that could be avoided if the splash screen load in full screen mode. 

For example,

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="AppTheme" parent="@android:style/Theme.Black.NoTitleBar">
        <item name="android:windowBackground">@drawable/splash</item>
        <item name="android:colorBackground">@color/black</item>
        <item name="android:windowFullscreen">true</item>
    </style>
</resources>
```

Ofcoz, it may need to set back to non-fullscreen for your application.

**Comment by redgrapetech** [comment]
thanks . will check it by making the app FS temporarily. 


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Ben Lau <notifications@github.com> Date: 5/8/17  16:57  (GMT+05:30) To: benlau/quickandroid <quickandroid@noreply.github.com> Cc: "Rajesh Mallah." <mallah@redgrape.tech>, Author <author@noreply.github.com> Subject: Re: [benlau/quickandroid] splash screen - minor issue (#38) 
yes, I also find this issue. I didn't go in deep to find out the main reason.  But that could be avoided if the splash screen load in full screen mode.
For example,
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="AppTheme" parent="@android:style/Theme.Black.NoTitleBar">
        <item name="android:windowBackground">@drawable/splash</item>
        <item name="android:colorBackground">@color/black</item>
        <item name="android:windowFullscreen">true</item>
    </style>
</resources>

Ofcoz, it may need to set back to non-fullscreen for your application.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/benlau/quickandroid","title":"benlau/quickandroid","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/benlau/quickandroid"}},"updates":{"snippets":[{"icon":"PERSON","message":"@benlau in #38: yes, I also find this issue. I didn't go in deep to find out the main reason.  But that could be avoided if the splash screen load in full screen mode. \r\n\r\nFor example,\r\n\r\n```\r\n\u003c?xml version=\"1.0\" encoding=\"utf-8\"?\u003e\r\n\u003cresources\u003e\r\n    \u003cstyle name=\"AppTheme\" parent=\"@android:style/Theme.Black.NoTitleBar\"\u003e\r\n        \u003citem name=\"android:windowBackground\"\u003e@drawable/splash\u003c/item\u003e\r\n        \u003citem name=\"android:colorBackground\"\u003e@color/black\u003c/item\u003e\r\n        \u003citem name=\"android:windowFullscreen\"\u003etrue\u003c/item\u003e\r\n    \u003c/style\u003e\r\n\u003c/resources\u003e\r\n```\r\n\r\nOfcoz, it may need to set back to non-fullscreen for your application.\r\n\r\n"}],"action":{"name":"View Issue","url":"https://github.com/benlau/quickandroid/issues/38#issuecomment-299842181"}}}
